includes/news/v2/mail/footer.php

```diff
@@ -1 +1,84 @@
-<h2>Footer</h2>
+
+<table class="row">
+	<tbody>
+		<tr>
+			<th class="small-12 large-12 columns first last">
+				<hr>
+			</th>
+		</tr>
+	</tbody>
+</table>
+
+<table class="row">
+	<tbody>
+		<tr>
+			<th class="small-12 large-12 columns first last">
+			    <center>
+					<a href="https://www.instagram.com/ucf.edu">
+						<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/social/instagram.png" alt="Instagram" align="center" class="float-center"
+							alt="UCF Tab" align="center" class="float-center" style="display:inline-block" width="32" height="32"></a>
+					<a href="https://www.twitter.com/UCF/">
+						<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/social/twitter.png" alt="Twitter" align="center" class="float-center"
+							alt="UCF Tab" align="center" class="float-center" style="display:inline-block" width="32" height="32"></a>
+					<a href="https://www.facebook.com/ucf/">
+						<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/social/facebook.png" alt="Facebook" align="center" class="float-center"
+							alt="UCF Tab" align="center" class="float-center" style="display:inline-block" width="32" height="32"></a>
+					<a href="https://www.youtube.com/user/UCF/">
+						<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/social/youtube.png" alt="YouTube" align="center" class="float-center"
+							alt="UCF Tab" align="center" class="float-center" style="display:inline-block" width="32" height="32"></a>
+				</center>
+			</th>
+		</tr>
+	</tbody>
+</table>
+
+
+<table class="row">
+	<tbody>
+		<tr>
+			<th class="small-12 large-12 columns first last float-center" align="center">
+				<center>
+					<img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/ucf-tab.png" width="150" height="202" alt="UCF Tab" align="center" class="float-center">
```

Comment:
We could link this tab to the website:
```suggestion
					<a href="https://www.ucf.edu"><img src="<?php echo bloginfo( 'stylesheet_directory' ); ?>/static/img/ucf-tab.png" width="150" height="202" alt="UCF Tab" align="center" class="float-center"></a>
```